client/galaxy/scripts/mvc/dataset/dataset-error.js

```diff
@@ -173,50 +170,26 @@ var View = Backbone.View.extend({
     /** Convert tab template */
     _getBugFormTemplate: function(dataset, job) {
         const Galaxy = getGalaxyInstance();
-        var inputs = [
-            {
-                help: _l("Your email address"),
-                options: [],
-                type: "text",
-                name: "email",
-                label: "Your email",
-                value: Galaxy.user.get("email")
-            },
-            {
-                help: _l(
-                    "Any additional comments you can provide regarding what you were doing at the time of the bug."
-                ),
-                options: [],
-                type: "text",
-                area: true,
-                name: "message",
-                label: "Message"
-            }
-        ];
-
-        // TODO
-        /*
-        if (false && response.any_public) {
-            inputs.push({
-                name: "public_consent",
-                label: "Public Disclosure Consent",
-                help:
-                    "This Galaxy is configured to report to one or more error reporting backends that public to the world. By selecting 'yes', you acknowledge that this bug report will be made public.",
-                value: String(Boolean(false)),
-                options: [],
-                type: "boolean"
-            });
-        }
-        */
-
-        var form = new Form({
-            title: _l("Error Report"),
-            inputs: inputs,
+        const form = new Form({
+            inputs: [
+                {
+                    type: "text",
+                    hidden: true,
```

Comment:
Can you make this hidden only if the user is not anonymous? Otherwise they won't be able to insert their email address to receive a reply.

I'd leave it not-hidden for all users, so one could choose a different email address to receive this kind of emails.

Reply:
I enabled the email for anonymous users although I am not sure how useful dataset error reporting is for an anon user, since the session session will be cleared.